release.toml

```diff
@@ -1,3 +1,3 @@
 pre-release-commit-message = "chore({{crate_name}}): release {{version}}"
-pro-release-commit-message = "chore({{crate_name}}): starting development cycle for {{next_version}}"
-no-dev-version = true
\ No newline at end of file
+post-release-commit-message = "chore({{crate_name}}): starting development cycle for {{next_version}}"
+dev-version = false
```

Comment:
I don't think we really use any of these. I'd remove the file and use the defaults (in the past `cargo-release` did create a new dev version, but they no longer do.

Reply:
done